md/dev/mediquette.txt

```diff
@@ -0,0 +1,74 @@
+
+Etiquette @medic
+
+Be kind to the plants and animals.
+
+Slack
+
+  1) Try to pay attention what channel you are writing in and keep noise
+     (tangential or unrelated discussion) to a minimum.  Use threads when
+     appropriate to keep tangential discussion contained and out of the main
+     channel.   Each channel should have a purpose with a description that
+     explains appropriate usage.
+
+  2) Never delete or edit someone else's content.  If you think something is
+     inappropriate then notify the person first and ask them politely to remove
+     it.
+
+  3) In general deleting something you already posted is poor etiquette.
+     Because it changes the context of the discussion.  If you have a problem
+     with something you wrote then just create a thread on it or just repost it
+     with your edits.
```

Comment:
I don't want to prohibit deletions or edits of peoples' own messages. There are perfectly valid reasons to do it  – e.g. you've posted a line of questions that turns out to be easily-answered, haven't seen any responses, and later discover it was obvious and would waste the time of others. Same thing for inaccurate information: you might want to edit it to ensure someone doesn't find it later and do the wrong thing. I'd probably outline what you consider good judgement rather than prohibiting use of a Slack feature.